protocols/imap/src/main/java/org/apache/james/imap/processor/base/SelectedMailboxImpl.java

```diff
@@ -411,7 +411,7 @@ public synchronized void event(Event event) {
 
```

Comment:
Oups it's look like there is a bug in GitHub this comment appear on two differents commit..